copyright

```diff
@@ -725,6 +724,12 @@ Copyright: Damien Jemison
 License: CC-BY-SA-3.0
 Comment: Taken from https://commons.wikimedia.org/wiki/File:Preamplifier_at_the_National_Ignition_Facility.jpg
 
+Files:
+ images/land/station15*
+Copyright: N/A (CC0 Public Domain)
+License: CC0
+Comment: Taken from https://pixabay.com/photos/step-industry-steel-3104846/ and cropped. According to Section 3 of the Pixabay Terms and Conditions, all images with a "published date" earlier than January 9, 2019 have the CC0 license.
```

Comment:
```suggestion
Comment:
 Taken from https://pixabay.com/photos/step-industry-steel-3104846/ and cropped.
 According to Section 3 of the Pixabay Terms and Conditions, all images with a
 "published date" earlier than January 9, 2019 have the CC0 license.
```
Match main repo

Reply:
Updated it to match other copyright styling.